src/kaocha/plugin/junit_xml.clj

```diff
@@ -143,12 +146,23 @@
   "Write test results to junit.xml"
 
   (cli-options [opts]
-    (conj opts [nil "--junit-xml-file FILENAME" "Save the test results to a Ant JUnit XML file."]))
+    (conj opts [nil
+                "--junit-xml-file FILENAME"
+                "Save the test results to a Ant JUnit XML file."
+
+                "--no-system-out"
+                "Will NOT print stdout into system-out tag"]))
```

Comment:
These options will be shown when people do `bin/kaocha --help` together with all other options from other plugins, so it's important to provide a bit of context.

```
"--junit-omit-system-out"
"Do not add captured output to junit.xml"
```